ontogpt-habitat/requirements.txt

```diff
@@ -0,0 +1,2 @@
+kafka-python==2.0.2
```

Comment:
use kafka-python-ng: https://stackoverflow.com/questions/77287622/modulenotfounderror-no-module-named-kafka-vendor-six-moves-in-dockerized-djan